plugin/resurrect/pane_tree.lua

```diff
@@ -68,6 +68,9 @@ end
 ---@param process string
 ---@return string?
 function pub.get_shell_process(process)
+	if not process or process == "" then
```

Comment:
Add a comment about this being because the process is in a ssh domain or similar. That would help future me understand this part :smile:

Reply:
I originally included this check to avoid the error related to concatenating a `nil` value when getting process from an SSH domain. We no longer call this function for SSH domains. As you point out in another comment, there are other potential types of domains (TLS, WSL, maybe others). I don't have a way to test whether these will return a `nil` or empty process, but this would prevent similar errors. I don't know if it's necessary and hopefully others can comment whether there is still a chance of a `nil` or empty string process value.